capreolus/evaluator.py

```diff
@@ -85,4 +127,6 @@ def search_best_run(runfile_dir, benchmark, metric, folds=None):
             best_scores["avg"] = {metric: np.mean(scores), "path": runfile}
 
     key = list(folds.keys())[0] if len(folds) == 1 else "avg"
-    return best_scores[key]
+    best_scores = best_scores[key]
+    best_scores[metric] = eval_runfile(best_scores["path"], benchmark.qrels, metric)[metric]
```

Comment:
I was returning the score calculated on dev only while actually it should be the score on the entire runfile? fixed here